enhancements/dns/externaldns-operator.md

```diff
@@ -235,15 +233,19 @@ such as a secret or configmap in an arbitrary namespace.
 By default, the operator would assume the cluster's platform is the desired DNS Provider. The operator would allow a user to specify
 an alternative External DNS Provider, such as BlueCat DNS. The ExternalDNS operator would be able to create multiple ExternalDNS deployments
 to run in concert, one for each ExternalDNS operator Custom Resource (similar to how the OpenShift [Ingress Operator](https://github.com/openshift/cluster-ingress-operator)
-consumes one or more Ingress Controller resources). The `domains` field would be used to filter source resources according to the requested DNS hostname
-(sans any domains matching any values specified in the `excludeDomains` field).
-If an ExternalDNS operator Custom Resource specifies multiple DNS Zones, multiple instances of ExternalDNS would be deployed within the same deployment accordingly.
+consumes one or more Ingress Controller resources). The `domains` field would be used to filter source resources based on the source resources desired hostname.
```

Comment:
Is the second "source resources" redundant, or is it supposed to be "source resources'" (possessive)?

Reply:
fixed.